monad-bayes.cabal

```diff
@@ -131,7 +136,7 @@ executable example
 
   if flag(dev)
     ghc-options:
-      -O2 -Wall -Wcompat -Wincomplete-record-updates
```

Comment:
Why remove O2?

Reply:
The package uploader suggested it makes no difference and slows down compilation times.